tests/integration/test_metrics.py

```diff
@@ -1880,3 +1878,172 @@ def test_metric_bucket_encoding_dynamic_global_config_option(
     else:
         assert metrics[dname]["value"] == [1337.0]
         assert metrics[sname]["value"] == [42.0]
+
+
+@pytest.mark.parametrize("is_processing_relay", (False, True))
+@pytest.mark.parametrize(
+    "global_generic_filters",
+    [
+        # Config is broken
+        {
+            "version": "Halp! I'm broken!",
+            "filters": [],
+        },
+        # Config is valid, but filters aren't supported
+        {
+            "version": "65535",
```

Comment:
Don't we expect an integer here?

```suggestion
            "version": 65535,
```